sigma/plugins.py

```diff
@@ -90,60 +150,74 @@ def _discover_module_directories(cls, module, directory_name : str, include : bo
         return result
 
     @classmethod
-    def autodiscover(cls, include_backends : bool = True, include_pipelines : bool = True, include_validators : bool = True):
```

Comment:
I ran `black` on this file only, which resulted in the changes from here on.

Reply:
The resulting code is much more readable, like it. But lets keep it back a while for the rest of the code. I'm currently implementing a new feature with some files touched and fear the merge conflicts 😉

Reply:
No rush!